src/atomic.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
Although a common `#pragma once` is not standard, so better to replace with `#ifndef ATOMIC_H`, `#define ATOMIC_H`, ..., `#endif` sequence (might use less generic guard macro than `ATOMIC_H`).

Also add the copyright boilerplate here.